packages/hooks/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
looks like we're missing a decent amount of properties here that probably should be brought in from the components package.json too

Reply:
Yep, we are, as I mentioned in the PR description, this was a fake package, a POC of sorts. 
> "Introduced a fake hooks package and used it in the component package as an example"